Added sechub client installation (#2746)
mercedes-benz/sechub: sechub-doc/src/docs/asciidoc/sechub-getting-started.adoc

Now you are ready to do scans!

==== Get SecHub client 

> The SecHub client is needed to initiate scans. In later sections of this tutorial, the client is used to do an example scan. In case you don't have it, you cannot continue with the tutorial. The command below, will download the latest version and put it in your `/usr/local/bin` folder.

Reviewer: ```suggestion
The SecHub client is needed to scan. In later sections of this guide, the client is used to scan an example. The command below, will download the latest version and put it in your `/usr/local/bin` folder.
```

[source, bash]
--
# Get latest version
CLIENT_VERSION=`curl -s https://mercedes-benz.github.io/sechub/latest/client-download.html | grep https://github.com/mercedes-benz/sechub/ | awk -F '-' '{print $NF}' | sed 's/.zip">//'`

# Download the zipped binary
curl -L -o sechub-cli.zip https://github.com/mercedes-benz/sechub/releases/download/v$CLIENT_VERSION-client/sechub-cli-$CLIENT_VERSION.zip

# Verify the binary
curl -L -o sechub-cli.zip.sha256 https://github.com/mercedes-benz/sechub/releases/download/v$CLIENT_VERSION-client/sechub-cli-$CLIENT_VERSION.zip.sha256
sha256sum --check sechub-cli.zip.sha256

# Extract 
unzip sechub-cli.zip

> sudo cp platform/linux-amd64/sechub /usr/local/bin

Reviewer: This is only the right command if you are using linux-amd64 (Intel or AMD 64-bit processors). Please add at least a command in the bash script that this might need to be changed for Intel 32-bit or ARM based CPUs.

}
--

==== Install SecHub's VSCodium Plugin (OPTIONAL)

> SecHub's VSCodium Plugin helps you better visualize where the vulnerabilities in your code are located.

Reviewer: ```suggestion
SecHub's VSCodium Plugin helps you to work faster with the SecHub report.
```
